packages/playground/mcp/README.md

```
@@ -86,6 +86,75 @@ The [Playground Website](https://playground.wordpress.net/) also supports [WebMC
+		description: 'Creates a draft order.',
+		inputSchema: { type: 'object', properties: { sku: { type: 'string' } } },
+		execute: async ( input ) => ( { orderId: await createOrder( input.sku ) } ),
+	} );
+	</script>
+	<?php
+} );
+```
+
+WordPress runs in a nested iframe an agent never sees, so Playground proxies these tools:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)